### ISSUE TITLE
语义分割训练自己数据，已经完成训练一轮，模型卡住。

### ISSUE THREAD

**Issue by yuxin7** [PaddlePaddle/PaddleX#1703]
训练已经完成，但不开启验证，莫名卡住。
![image](https://github.com/PaddlePaddle/PaddleX/assets/35183358/922cc1cf-463a-4d53-8fc9-04e04da14116)

**Comment by lailuboy** [comment]
pip list看下环境上的包版本，另外环境的操作系统和cuda版本信息发一下看看

**Comment by yuxin7** [comment]
> pip list看下环境上的包版本，另外环境的操作系统和cuda版本信息发一下看看

操作系统：win10
cuda：
![image](https://github.com/PaddlePaddle/PaddleX/assets/35183358/41fc31c9-5e04-46ee-a191-48c100c9eafd)

Package            Version
------------------ -------------
aiohttp            3.8.1
aiosignal          1.2.0
astor              0.8.1
async-timeout      4.0.2
asynctest          0.13.0
attrs              21.4.0
Babel              2.10.3
bce-python-sdk     0.8.73
certifi            2022.6.15.2
cfgv               3.3.1
chardet            5.0.0
charset-normalizer 2.1.1
click              8.1.3
cma                3.2.2
colorama           0.4.5
colorlog           6.7.0
cycler             0.11.0
Cython             0.29.32
datasets           2.3.2
decorator          5.1.1
dill               0.3.4
distlib            0.3.6
djl-python         0.19.0
easydict           1.9
filelock           3.7.1
flake8             4.0.1
Flask              2.1.3
Flask-Babel        2.0.0
Flask-Cors         3.0.10
fonttools          4.37.1
frozenlist         1.3.0
fsspec             2022.8.2
future             0.18.2
gast               0.3.3
GDAL               3.0.4
huggingface-hub    0.9.1
identify           2.5.5
idna               3.4
importlib-metadata 4.2.0
install            1.3.5
itsdangerous       2.1.2
jieba              0.42.1
Jinja2             3.1.2
joblib             1.1.0
kiwisolver         1.4.4
MarkupSafe         2.1.1
matplotlib         3.5.3
mccabe             0.6.1
multidict          6.0.2
multiprocess       0.70.12.2
nltk               3.7
nodeenv            1.7.0
numpy              1.21.6
onnx               1.12.0
opencv-python      4.6.0.66
opt-einsum         3.3.0
packaging          21.3
paddle-bfloat      0.1.7
paddle2onnx        1.0.0
paddlefsl          1.1.0
paddlehub          1.8.2
paddlenlp          2.4.0
paddlepaddle-gpu   2.0.2.post110
paddleslim         1.1.1
paddlex            1.2.6
pandas             1.1.5
Pillow             9.2.0
pip                22.2.2
platformdirs       2.5.2
pre-commit         2.20.0
protobuf           3.20.0
psutil             5.9.2
pyarrow            8.0.0
pycocotools        2.0.4
pycodestyle        2.8.0
pycryptodome       3.15.0
pyflakes           2.4.0
pyparsing          3.0.9
python-dateutil    2.8.2
pytz               2022.1
PyYAML             6.0
pyzmq              23.2.1
rarfile            4.0
regex              2022.9.13
requests           2.28.1
responses          0.18.0
scikit-learn       1.0.2
scipy              1.7.3
sentencepiece      0.1.96
seqeval            1.2.2
setuptools         65.3.0
shapely            2.0a1
six                1.16.0
sklearn            0.0
threadpoolctl      3.1.0
toml               0.10.2
torch              1.9.0+cu111
torchaudio         0.9.0
torchvision        0.10.0+cu111
tqdm               4.64.1
typing_extensions  4.3.0
urllib3            1.26.12
virtualenv         20.16.2
visualdl           2.4.0
Werkzeug           2.1.2
wheel              0.37.1
xxhash             3.0.0
yapf               0.26.0
yarl               1.7.2
zipp               3.8.1

**Comment by yuxin7** [comment]
但是训练样例是没有问题的

**Comment by yuxin7** [comment]
> pip list to see the package version on the environment, and send the operating system and CUDA version information of the environment to see

但是训练样例是没问题的

**Comment by lailuboy** [comment]
可能和数据集有关系，数据集是怎么标注的
